File: ce/snippets/visualbasic/CRMV8/exportribbonxmlvb/vb/exportribbonxml3.vb
' <snippetexportribbonxml3>


 'Retrieve the Application Ribbon
 Dim appribReq As New RetrieveApplicationRibbonRequest()
 Dim appribResp As RetrieveApplicationRibbonResponse = CType(_serviceProxy.Execute(appribReq), RetrieveApplicationRibbonResponse)

 Dim applicationRibbonPath As String = Path.GetFullPath(exportFolder &amp; "\applicationRibbon.xml")
 File.WriteAllBytes(applicationRibbonPath, unzipRibbon(appribResp.CompressedApplicationRibbonXml))

' </snippetexportribbonxml3>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/exportribbonxml/cs/exportribbonxml3.cs
// <snippetexportribbonxml3>


 //Retrieve the Application Ribbon
 RetrieveApplicationRibbonRequest appribReq = new RetrieveApplicationRibbonRequest();
 RetrieveApplicationRibbonResponse appribResp = (RetrieveApplicationRibbonResponse)_serviceProxy.Execute(appribReq);

 System.String applicationRibbonPath = Path.GetFullPath(exportFolder + "\\applicationRibbon.xml");
 File.WriteAllBytes(applicationRibbonPath, unzipRibbon(appribResp.CompressedApplicationRibbonXml));

// </snippetexportribbonxml3>